### DISCUSSION TITLE
posts/sejam-bem-vindos/

### DISCUSSION THREAD

**Discussion by giscus** [gianlcl/iglabs-blog#2] [Announcements]
# blog/0001-bemvindo/

Bom, primeiramente eu acho que devo me apresentar, embora eu fale um pouco sobre mim no Sobre, vou tentar falar um pouco diferente aqui. Para quem não me conhece, meu nome é Gian e na data dessa publicação falta exatamente um mês para meu aniversário de 30 anos. Eu trabalho com devops desde 2019 e atualmente estou desempenhando o papel de Platform Engineer na Magalu Cloud.
Escrever um blog é um projeto que eu adiava desde 2019, quando pela primeira vez descobri o que significava devops e comecei a estudar sobre a área. O meu objetivo com esse blog é criar um registro dos meus estudos para consulta do Gian do futuro, mas por tabela também compartilhar o que venho aprendendo com outras pessoas e quem sabe gerar alguma discussão sobre os assuntos.

https://blog.iglabs.com.br/blog/0001-bemvindo/

**Comment by nataliagranato** [comment]
Esperando por isso a muito tempo.
